### ISSUE TITLE
Table *blocklists or *blacklists? Sql defs don't match db_schema.json

### ISSUE THREAD

**Issue by chrisinmtown** [MISP/MISP#6540]
This is my attempt to report an apparent inconsistency in table name definitions.  Commt 12efbf9d renamed tables in db_schema.json, for example event_blacklist to event_blocklist. At the tip of branch 2.4, in file db_schema.json I see this:
```
        "event_blocklists": [
            {
                "column_name": "id",
                "is_nullable": "NO",
                "data_type": "int",
                "character_maximum_length": null,
                "numeric_precision": "10",
                "collation_name": null,
                "column_type": "int(11)",
                "column_default": null,
                "extra": "auto_increment"
            },
  ...
```
But I still see the old table names elsewhere, maybe some places were not (yet?) updated.   Again checking the tip of branch 2.4, I see these table definitions in DDL files in the INSTALL directory:
```
MYSQL.sql:CREATE TABLE IF NOT EXISTS `event_blacklists` (

POSTGRESQL-structure.sql:CREATE TABLE public.event_blacklists (
```
Please advise, thanks.

**Comment by mokaddem** [comment]
These were just leftovers that were converted upon the first successful login in MISP (as it runs all pending updates), so nothing to worry about.
I changed these schemas in #6556 to avoid confusion in the future.
Thanks for the report!

**Comment by chrisinmtown** [comment]
So this is fixed by PR https://github.com/MISP/MISP/pull/6556  ?  That's great, thank you.  Please close this when that is merged.

**Comment by mokaddem** [comment]
> So this is fixed by PR #6556 ? That's great, thank you. Please close this when that is merged.

Exactly and yes I will!

**Comment by iglocska** [comment]
This should be fixed now.
